packages/datadog-plugin-aws-sdk/test/integration-test/client.spec.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const {
+  FakeAgent,
+  createSandbox,
+  checkSpansForServiceName,
+  skipUnsupportedNodeVersions,
+  spawnPluginIntegrationTestProc
+} = require('../../../../integration-tests/helpers')
+const { assert } = require('chai')
+
+const describe = skipUnsupportedNodeVersions()
+
+describe('esm', () => {
+  let agent
+  let proc
+  let sandbox
+
+  before(async function () {
+    this.timeout(20000)
+    sandbox = await createSandbox(['aws-sdk'], false, [
+      `./packages/datadog-plugin-aws-sdk/test/integration-test/*`])
+  })
+
+  after(async () => {
+    await sandbox.remove()
+  })
+
+  beforeEach(async () => {
+    agent = await new FakeAgent().start()
+  })
+
+  afterEach(async () => {
+    proc && proc.kill()
+    await agent.stop()
+  })
+
+  context('aws-sdk', () => {
+    it('is instrumented', async () => {
+      const res = agent.assertMessageReceived(({ headers, payload }) => {
+        assert.propertyVal(headers, 'host', `127.0.0.1:${agent.port}`)
+        assert.isArray(payload)
+        assert.strictEqual(checkSpansForServiceName(payload, 'aws.request'), true)
+      }, undefined)
+
+      proc = await spawnPluginIntegrationTestProc(sandbox.folder, 'server.mjs', agent.port, undefined,
```

Comment:
(Note that here is is likely you might need to keep the undefined since there's an argument which follows.)